zebra-chain/src/sapling/tests.rs

```diff
@@ -1,2 +1,4 @@
 mod preallocate;
 mod prop;
+mod test_vectors;
+mod vectors;
```

Comment:
Same here, maybe vectors.rs should be tree.rs or something

Reply:
Good idea, renamed in 97ec26174